README.md

```diff
@@ -119,7 +119,7 @@ A schema is basically a single data structure. A keyword list, containing two to
     ],
     "my_app.complex_list.*": [
       to: "my_app.complex_list",
-      datatype: [:complex],
+      datatype: :complex,
```

Comment:
Without the list, it implies a single element complex type, rather than a list of complex type, was that intentional?